lib/crypto/ecdsa.js

```diff
@@ -10,6 +10,13 @@ var BufferUtil = require('../util/buffer');
 var _ = require('lodash');
 var $ = require('../util/preconditions');
 
+var secp256k1 = null;
+try {
+  secp256k1 = require('secp' + '256k1');
```

Comment:
Just tried running this with this build: https://travis-ci.org/bitpay/bitcore-lib/builds/160276007 and there still seems to be an issue with browserify.